com.microsoft.java.test.runner/src/main/java/com/microsoft/java/test/runner/Pair.java

```diff
@@ -0,0 +1,11 @@
+package com.microsoft.java.test.runner;
+
+public class Pair {

```

Comment:
`final` class?

Reply:
Could use this https://commons.apache.org/proper/commons-lang/apidocs/org/apache/commons/lang3/tuple/package-summary.html class since our binary is already depends on commons package